foundation/foundation-mda/src/main/resources/templates/deployment/spark-operator/spark-application-base-values.yaml.vm

```diff
@@ -103,6 +113,16 @@ sparkApp:
       env:
         - name: KRAUSENING_BASE
           value: /opt/spark/krausening/base
+        #if (${hasFileStore})
```

Comment:
Q/S: Do the executors need this? I'd expect that the actual accessing of the file store would be solely through the driver.

Reply:
Considering Drivers to spark applications are 1-to-1 and Executors to spark applications can be Many-to-1 I've removed the executor change and tested and it's not necessary.